test/phonons.jl

```diff
@@ -105,4 +121,72 @@ end
     end
 end
 
+# Return the torus for x ∈ [-½, ½) instead of [0, 1).
```

Comment:
As in previous PRs, not reviewing the test as I assume it's temporary, feel free to ping if you want a review on a specific part